cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,22 @@
+class PaymentStepPage {
+    private payButton: string;
+    private confirmOrder: string;
+
+    constructor() {
+        this.payButton = "a.bankwire";
+        this.confirmOrder = "#cart_navigation>button>span";
+    }
+
+    public confirmOrderm(): void {
```

Comment:
typo in:
```suggestion
    public confirmOrder(): void {
```

Reply:
i cannot change because create conflict with the name of the attribute . @danielgalvis98